### PR TITLE
Add wg-parallel-rustc.

### PR DESCRIPTION
**Pull request by michaelwoerister**
cc @Zoxc @nikomatsakis @davidtwco

### CODE REVIEW

File: working-groups/parallel-rustc/NOTES.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#### *<p align="center">This page is an example template. No meetings yet.</p>*
+
+# parallel-rustc Meeting Notes
+This document contains meeting notes from the NLL working group.
```
Comment (davidtwco) [inline working-groups/parallel-rustc/NOTES.md]:
```suggestion
This document contains meeting notes from the parallel rustc working group.
```

File: working-groups/parallel-rustc/NOTES.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+## 2019-02-22: [Triage Meeting][triage20190222]
```
Comment (davidtwco) [inline working-groups/parallel-rustc/NOTES.md]: Perhaps remove the link until there is a meeting to link to.

File: working-groups/parallel-rustc/NOTES.md (hunk 1 above)
Comment (nikomatsakis) [inline working-groups/parallel-rustc/NOTES.md]: NLL?

File: working-groups/parallel-rustc/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+## What if I don't have much time?
+If you don't have time to contribute code but you do compile your own compiler,
+the consider setting `parallel-compiler = true` in your `config.toml`. If you
+find a bug, we'd love it if you could file an issue.
+
+## Are there any resources so I can get up to speed?
+There are some resources available for those interested in contributing to get some background
+and context:
+
+- Zoxc's post in internals: https://internals.rust-lang.org/t/parallelizing-rustc-using-rayon/6606
```
Comment (davidtwco) [inline working-groups/parallel-rustc/README.md]: nit: inline link this so it is consistent with other links.

File: working-groups/parallel-rustc/NOTES.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#### *<p align="center">This page is an example template. No meetings yet.</p>*
```
Comment (nikomatsakis) [inline working-groups/parallel-rustc/NOTES.md]: I'd probably remove this and just say something like "This working group hasn't had any meetings yet."

File: working-groups/parallel-rustc/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+[michaelwoerister]: https://github.com/michaelwoerister
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Finish, optimize, and test the implementation of parallel compilation to an extent
+  that allows making it the default (and thus remove `cfg!(parallel_compiler)`).
+- Produce solid documentation of how parallel evaluation works, how it is
+  implementated, and what things it might affect in the [rustc-guide][rustc-guide].
```
Comment (nikomatsakis) [inline working-groups/parallel-rustc/README.md]: 🎉

File: working-groups/parallel-rustc/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+# parallel-rustc Working Group
```
Comment (davidtwco) [inline working-groups/parallel-rustc/README.md]: Can you also add this to the index table of the repo?

I'm going to add that to the instructions in the template as it has happened a few times and is a failing of the template.

**Comment by michaelwoerister** [comment]
Nits addressed. That markdown table in the top-level readme is not pretty `:)`

**Comment by nikomatsakis** [comment]
> Nits addressed. That markdown table in the top-level readme is not pretty 

yeah it's a pain

File: working-groups/parallel-rustc/NOTES.md (hunk 4 above)
Comment (davidtwco) [reply working-groups/parallel-rustc/NOTES.md]: This line from the template still exists.

Comment (michaelwoerister) [reply working-groups/parallel-rustc/NOTES.md]: Yeah, I thought it kinda makes sense to leave it like this until there are actual meeting notes.

Comment (davidtwco) [reply working-groups/parallel-rustc/NOTES.md]: Ah, I hadn't noticed that it changed to remove the "this page is an example template" part of the message. My bad.

**Pull request merged**